<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trainerClass</class>
 <widget class="QMainWindow" name="trainerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trainer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>631</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_collect">
     <attribute name="title">
      <string>Collect Data</string>
     </attribute>
     <widget class="QPushButton" name="btnPauseAVI">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>250</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>||</string>
      </property>
     </widget>
     <widget class="VideoGLWidget" name="videoWidget" native="true">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>320</width>
        <height>240</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>320</width>
        <height>100</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>Batch collecting information</string>
      </property>
      <widget class="QLabel" name="lbl1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Current Label:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Current Source:</string>
       </property>
      </widget>
      <widget class="QLabel" name="Avi">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>FPS:</string>
       </property>
      </widget>
      <widget class="QLabel" name="fps">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>271</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Collect samples from AVI</string>
      </property>
      <widget class="QListWidget" name="lstAVI">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRemoveAVI">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>21</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAddAVI">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add AVI</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCollectSamplesAVI">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Collect samples from AVI's</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>271</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Test decision function with Camera</string>
      </property>
      <widget class="QPushButton" name="btnLiveTest">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>250</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Live test</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_supervise">
     <attribute name="title">
      <string>Supervise Samples</string>
     </attribute>
     <widget class="QScrollArea" name="scroll">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>10</y>
        <width>404</width>
        <height>581</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>402</width>
         <height>579</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblSamplesGrid">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>371</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpDisplayInfo">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Display</string>
      </property>
      <widget class="QComboBox" name="cmbSampleFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter samples:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkLabelFilter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>32</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkUseCenter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string> Hand center as origin</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpSampleInfos">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>300</y>
        <width>201</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Sample Information</string>
      </property>
      <widget class="QListWidget" name="lstLabels">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>181</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPreviousSample">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnNextSample">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAddLabel">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>180</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblSample">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sample n/N</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>64</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Selected sample label:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRemoveLabel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>180</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRemoveSample">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>85</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>85</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>117</red>
             <green>116</green>
             <blue>118</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>117</red>
             <green>116</green>
             <blue>118</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Remove sample</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="grpLibraryInfo">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>550</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Library</string>
      </property>
      <widget class="QPushButton" name="btnLoadLibrary">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Load...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSaveLibrary">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save...</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblLibrary">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>[library not saved]</string>
       </property>
      </widget>
     </widget>
     <widget class="QSlider" name="sldSampleZoom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>600</y>
        <width>380</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>32</number>
      </property>
      <property name="maximum">
       <number>96</number>
      </property>
      <property name="singleStep">
       <number>16</number>
      </property>
      <property name="pageStep">
       <number>64</number>
      </property>
      <property name="value">
       <number>64</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>201</width>
        <height>190</height>
       </rect>
      </property>
      <property name="title">
       <string>Cumulative sample view</string>
      </property>
      <widget class="QLabel" name="lblCumulative">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>160</width>
         <height>160</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Train Samples</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>611</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Training</string>
      </property>
      <widget class="QPushButton" name="btnTrain">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Train</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="accuracy">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>171</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollLog">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>15</y>
         <width>331</width>
         <height>91</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>329</width>
          <height>89</height>
         </rect>
        </property>
        <widget class="QLabel" name="log">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>301</width>
           <height>81</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::LogText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>611</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Decision function</string>
      </property>
      <widget class="QPushButton" name="btnSaveDecisionFunction">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>16</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLoadDecisionFunction">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>16</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Load...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>401</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>[not yet trained]</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoGLWidget</class>
   <extends>QWidget</extends>
   <header>videoglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="trainer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chkLabelFilter</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbSampleFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
